src/utils.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports =
+  strContains: (str, searchValue) ->
+    str && str.toLowerCase().indexOf(searchValue) > -1
```

Comment:
You could have used [lodash's #contains/#include](https://lodash.com/docs/2.4.2#contains) for that instead of creating a new method


It seems that lodash is currently not a direct dependency, but as this project already depends on `br-tq1-helpers`, which has lodash's 2.x as a dependency, it could be easily added.